bigtable/google/cloud/bigtable/client.py

```diff
@@ -39,7 +39,12 @@
 
 from google.cloud.client import ClientWithProject
 
+from google.cloud.bigtable_admin_v2 import enums
 
+
+INSTANCE_TYPE_PRODUCTION = enums.Instance.Type.PRODUCTION
+INSTANCE_TYPE_DEVELOPMENT = enums.Instance.Type.DEVELOPMENT
+INSTANCE_TYPE_UNSPECIFIED = enums.Instance.Type.TYPE_UNSPECIFIED
```

Comment:
If we are going to expose these as public values, then we should likely add an `enums` module (e.g., `google.cloud.bigquery_admin_v2.enums`), import them there, and refer to that location in the docstrings.

Reply:
Added `enums` module to the current library with `InstanceType` and `StorageType` consts.